### ISSUE TITLE
Does this work for python?

### ISSUE THREAD

**Issue by Ridhwanluthra** [fkie/message_filters#1]
If yes, then how?
if no, what do I need to do to get it working with python?

**Comment by roehling** [comment]
No, the library is pure C++ at the moment. It would require either a reimplementation in Python, or some additional glue to expose the C++ classes in Python.
